examples/ref-implementation/coredns/manifests/cm-coredns.yaml

```diff
@@ -31,3 +31,9 @@ data:
         reload
         loadbalance
     }
+
+    localhost.localstack.cloud:53 {
+        errors
+        cache 30
+        forward . 10.96.100.10
```

Comment:
Possible to re-write instead of pointing to a static IP?

Reply:
https://kubernetes.io/docs/tasks/administer-cluster/dns-custom-nameservers/#configuration-of-stub-domain-and-upstream-nameserver-using-coredns

see: 

```
Note:
CoreDNS does not support FQDNs for stub-domains and nameservers (eg: "ns.foo.com"). During translation, all FQDN nameservers will be omitted from the CoreDNS config.
```

I'd rather point at the service, but that's apparently not an option, which is why I went the ClusterIP route.